app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user && user.authenticate(params[:session][:password])
```

Comment:
Usually preferable to define strong params somewhere below, ex:
```ruby
private

def session_params
  params.require(:session).permit(:email, :password, :remember_me)
end
```
and access that instead of raw `params` to enforce required/allowed params